core/schema/container.go

```diff
@@ -419,6 +419,11 @@ func (s *containerSchema) Install() {
 			Doc(`The error stream of the last executed command.`,
 				`Will execute default command if none is set, or error if there's no default.`),
 
+		dagql.Func("withAnnotation", s.withAnnotation).
```

Comment:
Can you add a `withoutAnnotation` as well?

Reply:
Done! PTAL